src/material-experimental/mdc-slider/_slider-theme.scss

```diff
@@ -1,22 +1,43 @@
-// TODO: disabled until we implement the new MDC slider.
-// @import '@material/slider/mixins.import';
+@import '@material/slider/slider';
+@import '@material/slider/slider-theme';
 @import '../mdc-helpers/mdc-helpers';
+@import '../../material/core/ripple/ripple';
 
 @mixin mat-mdc-slider-color($config-or-theme) {
   $config: mat-get-color-config($config-or-theme);
   @include mat-using-mdc-theme($config) {
-    // TODO: disabled until we implement the new MDC slider.
-    // @include mdc-slider-core-styles($query: $mat-theme-styles-query);
+    @include without-ripple($query: $mat-theme-styles-query);
 
     .mat-mdc-slider {
+      &.mat-primary, &.mat-accent, &.mat-warn {
+        @include value-indicator-color(
+          $color: #000,
+          $opacity: 0.6,
+          $query: $mat-theme-styles-query
+        );
+        @include value-indicator-text-color(
+          $color: on-primary,
```

Comment:
This doesn't seem right, I would expect `.mat-accent` to use `on-secondary` and `.mat-warn` to use `on-error`

Reply:
@mmalerba Is this comment resolved?

Reply:
I think the tick mark active color is still wrong. This is setting it to on-primary for the accent and warn sliders. You probably need to move that part into the helper mixin